File: 1-programming/3-control-flow/main.java
// main.java
// Java examples demonstrating control flow statements.

class CheckNumber {

    public static void checkNumber(int num) {
        if (num > 0) {
            System.out.println(num + " is Positive");
        } else if (num == 0) {
            System.out.println(num + " is Zero");
        } else {
            System.out.println(num + " is Negative");
        }
    }

    public static void main(String[] args) {
        for (int i = -1; i <= 1; i++) {
            checkNumber(i);
        }
    }
}

File: 1-programming/3-control-flow/main.cpp
// main.cpp
// C++ examples demonstrating control flow statements.

#include <iostream>

void checkNumber(int num) {
    if (num > 0) {
        std::cout << num << " is Positive" << std::endl;
    } else if (num == 0) {
        std::cout << num << " is Zero" << std::endl;
    } else {
        std::cout << num << " is Negative" << std::endl;
    }
}

int main() {
    for (int i = -1; i <= 1; i++) {
        checkNumber(i);
    }
    return 0;
}
